include/fmt/format.h

```diff
@@ -2703,15 +2703,19 @@ template <typename Range> class basic_writer {
     write_int(value, s);
   }
 
-  void write(double value) { write_double(value, format_specs()); }
+  void write(double value, const format_specs& spec = format_specs()) {
```

Comment:
Can `format_specs()` be replaced with `{}` or will gcc 4.4 complain?

Reply:
Fixed. This compiles with gcc 4.4

Reply:
I meant dropping the type name altogether and just using braces as in ` = {}`, not to repeat the argument type again.

Reply:
Sorry for misunderstanding. Unfortunately `= {}` not works for gcc 4.4 so lets currently keep the solution with `= format_specs()`